tests/ci/run_cryptofuzz.sh

```diff
@@ -67,12 +73,18 @@ SRC_CORPUS="$CRYPTOFUZZ_SEED_CORPUS"
 # 5 minutes for building AWS-LC and Cryptofuzz
 # 16 minutes (1000 seconds) of fuzzing
 # 24 minutes of cleanup and merging in new inputs
-TIME_FOR_EACH_FUZZ=1000
+TIME_FOR_EACH_FUZZ=${TIME_FOR_EACH_FUZZ:=1000}
 
 # Some fuzz tests can take a while but still pass. This is a tradeoff: less false positive noise, but some inputs that take
 # a long time could lead to a denial of service avenue. We're mostly interested in correctness and memory safety at this
 # time so we're willing to take the fit on fuzz speed
-FUZZ_TEST_TIMEOUT=30
+FUZZ_TEST_TIMEOUT=${FUZZ_TEST_TIMEOUT:=30}
+
+# Cryptofuzz lets us specify the operations and algorithms to fuzz
+# We use this to narrow the fuzzing run to only what's supported by the Java/ACCP module rather than all of OpenSSL
+FUZZ_TEST_ADDITIONAL_ARGS="--operations=Digest,HMAC,ECDSA_Verify"
+FUZZ_TEST_ADDITIONAL_ARGS="${FUZZ_TEST_ADDITIONAL_ARGS} --digests=NULL,MD5,SHA1,SHA256,SHA384,SHA512"
+FUZZ_TEST_ADDITIONAL_ARGS="${FUZZ_TEST_ADDITIONAL_ARGS} --curves=secp192r1,secp256r1,secp384r1,secp256k1"
```

Comment:
i don't think we [support](https://github.com/search?q=repo%3Acorretto%2Famazon-corretto-crypto-provider+secp192r1&type=code) secp192r1 anymore. suggest replacing that with secp521r1
```suggestion
FUZZ_TEST_ADDITIONAL_ARGS="${FUZZ_TEST_ADDITIONAL_ARGS} --curves=secp521r1,secp256r1,secp384r1,secp256k1"
```

Reply:
Anything we specify here needs to be the intersection of what we support and what [cryptofuzz supports](https://github.com/guidovranken/cryptofuzz/blob/master/modules/java/module.cpp#L390). Would be easy enough to develop, but that's a different project around uplifting their operation/algorithm support.

Reply:
gotcha. but we don't support secp192r1 so can we drop it here?